sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -7241,7 +7241,7 @@ atomic_update(AccessorTy acc, simd<Toffset, N> byte_offset, simd_mask<N> mask,
 }
 
 /// simd<T, N>
-/// atomic_update(AccessorT acc, simd<OffsetObjT, N> byte_offset,
+/// atomic_update(AccessorT acc, simd<OffsetSimdViewT, N> byte_offset,
```

Comment:
this line should not be changed, see L7272 below.

Reply:
ah wow the comment was wrong, good catch